doc/classes/CollisionObject2D.xml

```diff
@@ -154,6 +155,7 @@
 			<argument index="1" name="shape_id" type="int">
 			</argument>
 			<description>
+				Returns the index of the shape at given [code]owner_id[/code] or -1 if nothing was found.
```

Comment:
Could probably rephrase it to make it clearer. I'd add a line to explain how/what you use this index for.